python/lsst/ts/scheduler/driver.py

```diff
@@ -711,17 +711,18 @@ def select_next_target(self):
             self.last_winner_target = self.nulltarget
         return self.last_winner_target
 
-    def register_observation(self, observation):
+    def register_observation(self, observation,isColdStart=False):
```

Comment:
Maybe the variable name should use `lower_case_with_underscores` (e.g. `cold_start` or `is_cold_start`) instead of `camelCase`? I think PEP8 suggests the first style and this is the one adopted on this module.